### ISSUE TITLE
CO only | leap in some metrics between 2021 and 2022

### ISSUE THREAD

**Issue by sbassett** [TNC-NMFO/NWLAND#126]
e.g. 
![image](https://user-images.githubusercontent.com/3913946/149857930-eada72bc-5e9c-4fa6-9ec0-8a933fe9caa4.png)

**Comment by sbassett** [comment]
also All_region_All_land_All_own_CO_MAX_v01_solelyCO_A_rcp45_climNA_output_mean_BC1_NR120_ghg_ann_comp_output.pdf
![image](https://user-images.githubusercontent.com/3913946/149859955-149f7239-3dbb-44fc-95be-c6a958715ce5.png)

**Comment by sbassett** [comment]
Leap/spike still present.
![image](https://user-images.githubusercontent.com/3913946/165164998-a259074d-5634-45d5-8846-2b2c7938d823.png)

**Comment by sbassett** [comment]
The spike is present in all Ambition scenarios so the "change from baseline" stats should all be valid.

This dramatic change netween 2021 and 2022 seems to stem from Developed_all:
outputs\output_co_rcp85\A_Historical_co_rcp85_climNA_output_mean_BC1_NR120.xls > All_orgC_den


All_region | Water | All_own | 101.62 | 100.22 | 98.86 | 97.49
-- | -- | -- | -- | -- | -- | --
All_region | Sparse | All_own | 231.88 | 230.26 | 228.67 | 227.07
All_region | Shrubland | All_own | 133.13 | 133.87 | 134.59 | 135.34
All_region | Grassland | All_own | 159.28 | 157.53 | 155.81 | 153.88
All_region | Forest | All_own | 232.87 | 237.35 | 241.77 | 246.24
All_region | Meadow | All_own | 135.21 | 136.48 | 137.73 | 139.06
All_region | Cultivated | All_own | 94.96 | 91.98 | 89.12 | 86.4
All_region | Developed_all | All_own | 98.71 | 84.76 | 85.02 | 85.29
All_region | Ice | All_own | 108.59 | 107.14 | 105.7 | 104.24
All_region | Barren | All_own | 146 | 145.93 | 145.87 | 145.8
All_region | Desert | All_own | 80.47 | 81.86 | 83.24 | 84.65
All_region | Woodland | All_own | 189.94 | 190.9 | 191.75 | 192.54

**Comment by sbassett** [comment]
Hypotheses:

- [x] Scenario File Broken [Maybe dead removal needs to have the same 2021-2030, 2031-2040, and 2041-2050 time frames?]
- [x]  Jumps in developed area from landuse change projections?

Scenarios file doesn't need to have Dead_removal time periods match the other practices. [per example CALAND scenarios file]

Developed_all doesn't change unexpectedly in area between 2021 and 2022.

**Comment by sbassett** [comment]
outputs\output_co_rcp85\A_Historical_co_rcp85_climNA_output_mean_BC1_NR120.xls > All_orgC_den

 Developed_all
Smallest changes between 2021 and 2022:
616002
516008
3316008
3916001
616007
3216005
LARGEST CHANGES in 2021-22: 
2516008
2216011
2216001
216011
6116007
3416008

**Comment by sbassett** [comment]
This is caused by the application of dead removal to a built up legacy pile of dead down and litter. To resolve, the litter and dead down are being removed from initial stock estimate sin developed_all land types/landCats.

**Comment by aj1s** [comment]
@sbassett - In our 4/25 runs, we had seen this dramatic first-year drop in both CO and NM.   As noted above, we removed all but a trace of C in the dead down (DDC) and litter (LTC) pools within the Developed_all land_type. (Actual summary stats for the respective landcats in stocks_ddc_co.csv, stocks_ddc_nm.csv, stocks_ltc_co.csv, and stocks_ltc_nm.csv were set as: min = 0, max = 0.000002, mean = 0.000001, stddev = 3.00E-15, and sum = non_null*mean.)  

This has resolved the first-year drop in the respective annual NM figs, e.g., .../output_nm_rcp85/mean/All_region/All_land/All_own/All_region_All_land_All_own_C_Potential_nm_rcp85_climNA_output_mean_BC1_NR120_ghg_ann_comp_output.pdf

![Y](https://user-images.githubusercontent.com/13356563/168217964-8ab1cb7f-6723-40a2-a0de-dee99d96b4da.JPG)

... but alas has not in corresponding CO figs, e.g., 
.../output_co_rcp85/mean/All_region/All_land/All_own/All_region_All_land_All_own_C_Potential_co_rcp85_climNA_output_mean_BC1_NR120_ghg_ann_comp_output.pdf

![X](https://user-images.githubusercontent.com/13356563/168218045-e87680ba-8393-49e1-b8af-c607c2882b33.JPG)

**Comment by sbassett** [comment]
@aj1s is this pattern solely present in the devloped_all (and all_land) landtype graphs?

Maybe output from an old 'pre-zeroed out DDC and LTC on delveoped_all' run is used as input for plot_caland()?

**Comment by aj1s** [comment]
@sbassett 
1/ yep -- evident only in developed_all and all_land for CO. However, the CO drop is not as dramatic as it appears at first; see respective magnitudes on the Y-axis, NM vs CO. 
2/ On your 2nd Q: I highly doubt it as I ran write_caland_inputs.r and CALAND.r anew with each run, but good Q and I'll double-check.

**Comment by sbassett** [comment]
![image](https://user-images.githubusercontent.com/3913946/168367563-57a913cb-4a96-4ca1-8ad0-93ebd63ad70c.png)
Input file still has 'non-basically zero' DDC an LTC values.

**Comment by aj1s** [comment]
Yes, I would've expected these to be lower. However, they are comparable to the corresponding numbers in the NM input files.
